rust/xaynet-client/src/mobile_client/participant/mod.rs

```diff
@@ -32,19 +34,93 @@ pub struct ParticipantState {
     pub keys: SigningKeyPair,
     // Mask config
     pub aggregation_config: AggregationConfig,
+    pub max_message_size: MaxMessageSize,
 }
 
 #[derive(Serialize, Deserialize)]
 pub struct ParticipantSettings {
     pub secret_key: ParticipantSecretKey,
     pub aggregation_config: AggregationConfig,
+    pub max_message_size: MaxMessageSize,
+}
+
+const MESSAGE_HEADER_LEN: usize = 136;
+const ENCRYPTION_HEADER_LEN: usize = 48;
```

Comment:
those constants are duplicates of `xaynet_core::message::message::HEADER_LENGTH` and `xaynet_core::crypto::encrpyt::SEALBYTES`, we should use the original ones instead of redefining them here.

Reply:
Good poitn